core/templates/components/stale-tab-info/stale-tab-info-modal.component.spec.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright 2022 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for stale tab information component.
+ */
+
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { StaleTabInfoModalComponent } from './stale-tab-info-modal.component';
+
+describe('Stale tab info modal component', () => {
+  let component: StaleTabInfoModalComponent;
+  let fixture: ComponentFixture<StaleTabInfoModalComponent>;
+  let ngbActiveModal: NgbActiveModal;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        StaleTabInfoModalComponent
+      ],
+      providers: [
+        NgbActiveModal
+      ]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(StaleTabInfoModalComponent);
+    component = fixture.componentInstance;
+    ngbActiveModal = TestBed.inject(NgbActiveModal);
+  });
+
+  it('should close modal', () => {
+    spyOn(ngbActiveModal, 'close');
+    component.closeModal();
+    expect(ngbActiveModal.close).toHaveBeenCalled();
```

Comment:
```suggestion
    spyOn(ngbActiveModal, 'close');

    component.closeModal();

    expect(ngbActiveModal.close).toHaveBeenCalled();
```

Reply:
I don't see the change.

Reply:
I forgot to push it at last. I have done it now.